src/NDItk/MultigroupTable/src/verify.hpp

```diff
@@ -6,16 +6,46 @@
  */
 void verify() {
 
+  // required records
+  if ( this->primaryGroupBoundaries().empty() ||
+       this->velocities().empty() ||
+       this->fluxWeights().empty() ||
+       this->reactionCrossSections().empty() ) {
```

Comment:
Did the document say reactionXS is required? I think we always have it, but PARTISN at least would run fine without any.

Reply:
That's what I think it says

Reply:
Once the entire table is done, we can go over the pdf and check exactly what is optional and what is required.